src/object/srv_obj_migrate.c

```diff
@@ -2070,8 +2070,7 @@ migrate_one_create(struct enum_unpack_arg *arg, struct dc_obj_enum_unpack_io *io
 		       DP_UUID(iter_arg->pool_uuid));
 		D_GOTO(put, rc = 0);
 	}
-	if (iod_eph_total == 0 || tls->mpt_version <= version ||
-	    tls->mpt_fini) {
+	if (iod_eph_total == 0 || tls->mpt_fini) {
```

Comment:
just confirm why remove "tls->mpt_version <= version" and what is it for in original code? thanks.

Reply:
Oh, see the commit message.  That rebuild fence check is for removing this mpt version check.  The purpose is 
1. removing the pool map version in recx to save some space.
2. To use epoch to tell the old and new data, instead of pool map, which would be convenient for vos_discard, once we need discard those half-reintegrated data.